source/CargoHold.cpp

```diff
@@ -38,7 +39,7 @@ namespace {
 		for(const auto &it : outfits)
 			sortedOutfits.emplace_back(it.first);
 
-		sort(sortedOutfits.begin(), sortedOutfits.end(),
+		sort(parallel::par_unseq, sortedOutfits.begin(), sortedOutfits.end(),
```

Comment:
again, probably unnecessary

Reply:
Sort should generally be using `unseq`, because simd sort can actually have really good performance, but we only get that option in C++20.

~~And yes, I did just slap this on because I could.~~